requirements.txt

```diff
@@ -1,2 +1,3 @@
 torch>=1.6
-pytorch-lightning>=1.1.1, <1.2
+torchvision>=0.8
```

Comment:
`torchvision` is currently an optional package for Bolts, so would you include the change in `requirements/models.txt`?:
https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/86e3f52308fa02e2a988b4977858a945f7d15ab2/requirements/models.txt#L1